tests/cases/fourslash/formatOnEnterOpenBraceAddNewLine.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="fourslash.ts"/>
+
+//// if(true) {/*0*/}
+//// if(false)/*1*/{
+//// }
+
+format.setOption("PlaceOpenBraceOnNewLineForControlBlocks", true);
+goTo.marker("0");
+edit.insertLine("");
+goTo.marker("1");
```

Comment:
Would it be interesting to check the file contents before inserting the second newline (to confirm that it's not formatting too much)?